src/NUnitConsole/nunit4-console/TestEventHandler.cs

```diff
@@ -1,12 +1,10 @@
 // Copyright (c) Charlie Poole, Rob Prouse and Contributors. MIT License - see LICENSE.txt
 
-using System;
 using System.Xml;
 using NUnit.Common;
 using NUnit.Engine;
-using NUnit.TextDisplay;
 
-namespace NUnit.ConsoleRunner
+namespace NUnit.TextDisplay
```

Comment:
You changed the namespace, but left the code in the nunit4-console project.

Reply:
I moved it and then put it back because I think each runner has to decide how to handle events. Will fix.